tigervnc: the string "-I" in path name is replaced wrongly

In the file unix/xserver/hw/vnc/Makefile.am, sed replaces all "-I"
string in the path. It should only replace the -I option used for
compiler.

Signed-off-by: Jian Liu <jian.liu@windriver.com>
---
 unix/xserver/hw/vnc/Makefile.am | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/unix/xserver/hw/vnc/Makefile.am b/unix/xserver/hw/vnc/Makefile.am
index 9885b8a..bff2302 100644
--- a/unix/xserver/hw/vnc/Makefile.am
+++ b/unix/xserver/hw/vnc/Makefile.am
@@ -68,7 +68,7 @@ fb.h: $(top_srcdir)/fb/fb.h
 pixman.h:
 	for i in ${XSERVERLIBS_CFLAGS}; do \
 		if [[ "$$i" =~ "pixman" ]]; then \
-			PIXMANINCDIR=`echo $$i | sed s/-I//g`; \
+			PIXMANINCDIR=`echo $$i | sed s/^ *-I//g`; \
 		fi; \
 	done; \
 	if [ ! "$$PIXMANINCDIR" = "" ]; then \
-- 
1.8.5.2.233.g932f7e4

